force-app/main/default/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -1374,377 +961,19 @@ public with sharing class AFFL_MultiRecordType_TEST {
-        String bizOrgContactFieldApiName = AFFL_MultiRecordType_TEST.getBusinessOrganizationFieldApiName();
-        String householdContactFieldLabel = AFFL_MultiRecordType_TEST.getHouseholdFieldLabel();
-
-        wrapperList.add(
-            new AfflMappingWrapperByNameAndApiName(
-                UTIL_Describe.getCustomHHAccRecTypeID(),
-                contactPrimaryAffiliationFieldApiNameList[0]
-            )
-        );
-
-        wrapperList.add(
-            new AfflMappingWrapperByNameAndLabel(
-                UTIL_Describe_API.getBizAccRecTypeID(),
-                contactPrimaryAffiliationFieldLabelList[1]
-            )
-        );
-
-        wrapperList = AFFL_MultiRecordType_TEST.commonSetup(wrapperList);
-        //Custom setup ended
-
-        List<Affl_Mappings__c> allMappings = [SELECT Id FROM Affl_Mappings__c];
-        System.assertEquals(wrapperList.size(), allMappings.size());
-
-        // Construct a list of primary Affiliation field Labels and API names
-        List<String> primaryFieldsList = new List<String>();
-        primaryFieldsList.add(bizOrgContactFieldApiName);
-        primaryFieldsList.add(householdContactFieldLabel);
-
-        Test.startTest();
-        AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
-        List<String> validPrimaryFieldList = afflMapper.getValidPrimaryAfflFields(primaryFieldsList);
-        Test.stopTest();
-
-        // Ensure List has all mappings
-        System.assertEquals(wrapperList.size(), validPrimaryFieldList.size());
-    }
-
-    // Positive test for getPrimaryAffiliationLookupAPI method of AFFL_MultiRecordTypeMapper using API name
-    @isTest
-    private static void getPrimaryAffiliationLookupAPIPoitiveAPITest() {
-        AFFL_MultiRecordType_TEST.setupMappingsByNameAndLabel();
-
-        String bizOrgContactFieldApiName = AFFL_MultiRecordType_TEST.getBusinessOrganizationFieldApiName();
-
-        Test.startTest();
-        AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
-        String primaryAfflLookupAPIName = afflMapper.getPrimaryAffiliationLookupAPI(bizOrgContactFieldApiName);
-        Test.stopTest();
-
-        System.assertEquals(bizOrgContactFieldApiName, primaryAfflLookupAPIName);
-    }
-
-    // Negative test for getPrimaryAffiliationLookupAPI method of AFFL_MultiRecordTypeMapper
-    @isTest
-    private static void getPrimaryAffiliationLookupAPINegativeTest() {
-        AFFL_MultiRecordType_TEST.setupMappingsByNameAndLabel();
-
-        Test.startTest();
-        AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
-        String primaryAfflLookupAPIName = afflMapper.getPrimaryAffiliationLookupAPI('Test1234');
-        Test.stopTest();
-
-        System.assertEquals(null, primaryAfflLookupAPIName);
-    }
-
-    /**************************************************************************
```

Comment:
Extracted from here to force-app/main/default/classes/AffiliationTestSetupFactory_TEST